index.html

```diff
@@ -12,16 +12,16 @@
 </head>
 
 <body>
-    <header class="header">
-        <nav class="nav">
-            <h3><a href="">Kigozi Joseph</a></h3>
-            <i class="nav-item fas fa-bars fa-lg"></i>
-        </nav>
-    </header>
     <div>
-
+        <header class="header">
+            <nav class="nav">
+                <h1><a href="">Kigozi Joseph</a></h1>
+                <i class="fas fa-bars fa-lg"></i>
+            </nav>
+        </header>
+        <main>
         <section class="catch-text-sec">
-            <h2 id="catch-text-heading">I'm Kigozi<br/> glad to see <br/> you!</h2>
+              <h2 id="catch-text-heading">I'm Kigozi<br/> glad to see <br/> you!</h2>
             <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
             <h3 class="socials-header">LET'S CONNECT</h3>
             <ul class="socials-list">
```

Comment:
Well done for implementing the changes requested by the previous reviewer. But it seems like you introduced a different design from the one provided by the Figma templates. Kindly ensure your design is close enough like the provided Figma design.

Your design |  Figma
---- | ----
![image](https://user-images.githubusercontent.com/84156714/205405157-09ed931c-c4d7-4f36-851f-7578f5df8083.png) | ![image](https://user-images.githubusercontent.com/84156714/205404910-103c7aef-d4f0-45b3-926d-d00148350b3c.png)

Reply:
@bobb-Rob  i mean the boarder radius?